lib/actions/lookupObject.js

```diff
@@ -11,7 +10,7 @@ const SALESFORCE_API_VERSION = process.env.SALESFORCE_API_VERSION || '45.0';
  */
 module.exports.getMetaModel = async function getMetaModel(configuration) {
   configuration.metaType = 'lookup';
-  const metaLoader = new MetaLoader(configuration);
+  const metaLoader = new MetaLoader(configuration, this);
   const metaData = await metaLoader.loadMetadata();
```

Comment:
Why we leave `await` here, but remove it in the `lookup.js`?

Reply:
Here we need `await` because right after `await` we have a code that use `metaData` variable.
In previous case we know that code calling our function expect that function asynchronous and know how to handle it.